### PR TITLE
Cecilie/js for loop experiment

### PR DESCRIPTION
**Pull request by cecilievag**

### CODE REVIEW

File: Nodejs/HttpTriggerIL/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    let x = 0, y = 0;
+
+    for (var i = 0; i < n; i++) {
+        x = 1 + 1;
+        y = y + x;
+    }
+}
+
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
```
Comment (kentis) [inline Nodejs/HttpTriggerIL/index.js:11]: This could be a source of noise since it involves I/O

File: Nodejs/HttpTriggerOL/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+function out_loop(n) {
+    let x = 1 +1, y = 0;
+
+    for (var i = 0; i < n; i++) {
+        y = y + x;
+    }
+}
+
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
```
Comment (kentis) [inline Nodejs/HttpTriggerOL/index.js:10]: potential source of noise?

Comment (cecilievag) [reply Nodejs/HttpTriggerOL/index.js]: Removed

File: Nodejs/HttpTriggerIL/index.js (hunk 1 above)
Comment (cecilievag) [reply Nodejs/HttpTriggerIL/index.js]: Removed

**Pull request merged**